server/utils/sentimentHandler.js

```diff
@@ -0,0 +1,15 @@
+const Sentiment = require("sentiment");
+
+var sentiment = new Sentiment();
+const sentimentHandler = (context) => {
+  const evaluation = sentiment.analyze(context).comparative;
+  if (evaluation > 0) {
+    return "positive";
+  } else if (evaluation < 0) {
+    return "negative";
+  } else {
+    return "neutral";
+  }
+};
```

Comment:
Looks good. Out of curiosity, does it just calculate it by itself? Or are there variables we can tweak that would affect the thresholds for certain sentiments?

Reply:
Yes, It's will calculate itself. The comparative has a range of -5 to 5, with 0 neutral. Basically, they have a  list of words with different scores. They will calculate by adding them together and divided by the total number of words. We can overwrite the score of words, or add the score to the word we want.